site/docs/api-quickstart.md

```diff
@@ -47,7 +47,7 @@ logsDF.write
 
 The logs [schema](#create-a-schema) and [partition spec](#create-a-partition-spec) are created below.
 
-### Using Hadoop tables
+### Using Apache Hadoop tables
```

Comment:
This is referring to file-system tables, which we call "Hadoop" tables, not to the Apache Hadoop project. Adding "Apache" here is misleading because we aren't talking about some concept of a table borrowed from Hadoop.